composer/algorithms/low_precision_layernorm/low_precision_layernorm.py

```diff
@@ -106,19 +103,20 @@ def apply_low_precision_layernorm(model, optimizers: Union[torch.optim.Optimizer
 
 class LowPrecisionLayerNorm(Algorithm):
     """
-    Replaces all instances of `torch.nn.LayerNorm` with `composer.algorithms.low_precision_layernorm.low_precision_layernorm.LPLayerNorm`.
+    Replaces all instances of :class:`torch.nn.LayerNorm` with class:`.LPLayerNorm`.
 
-    LPLayerNorm is a thin wrapper around `torch.nn.LayerNorm` which forces the layer to run in lower precision (torch.float16 or torch.bfloat16)
-    if autocast is enabled. This algorithm has no effect in FP32 or DeepSpeed FP16 mode, where autocast is disabled.
+    LPLayerNorm is a thin wrapper around :class:`torch.nn.LayerNorm` which forces the layer to run
+    in lower precision (torch.float16 or torch.bfloat16) if autocast is enabled. This algorithm has
+    no effect in FP32 or DeepSpeed FP16 mode, where autocast is disabled.
 
     This algorithm is intended to be used instead of Fused LayerNorm. They have similar behavior and performance.
 
     Args:
         apply_at (Event, optional): Event where algorithm is applied.
     """
 
-    def __init__(self, apply_at: Optional[Event] = None):
-        self.apply_at = Event.INIT if apply_at is None else apply_at
+    def __init__(self, apply_at: Optional[Event] = Event.INIT):
```

Comment:
Do we actually want this to be Optional? Is there a use case for passing `None`?